collections/reports/defaultannotationsword.php

```diff
@@ -13,7 +13,7 @@
 $lFooter = $_POST['lfooter'];
 $detIdArr = $_POST['detid'];
 $action = array_key_exists('submitaction',$_POST)?$_POST['submitaction']:'';
-$rowsPerPage = array_key_exists('rowcount',$_POST)?$_POST['rowcount']:3;
+$rowsPerPage = array_key_exists('columncount',$_POST)?$_POST['columncount']:3;
```

Comment:
Should the $rowsPerPage variable also be renamed to avoid confusion?

Reply:
Fixed